Object Analysis.ipynb

```diff
@@ -17,7 +17,7 @@
     "2. Run all of the cells\n",
     "\n",
     "3. Look at the summary pdf generated AND/OR explore each metric below.\n",
-    "    - Under each Metric there will be a portion of \"Setup\" and then \"Analyses\". Ignore the \"Setup\" unless customization is needed, and in \"Analyses\" results are shown to be interacted with. The number that comes after the M in the title refers to the measurement number when collecting the metrics.\n",
+    "    - All measurement metrics are identified by a number from 0-11. Each metric consists of a \"Setup\" portion and an \"Analyses\" portion. The \"Setup\" portion contains code that does not need to be modified unless customization is needed, and the \"Analyses\" portion provides an interactive display of the results. \n",
```

Comment:
looks good! please add this same change to the gender, attribute, and geography notebooks